src/unistd.rs

```diff
@@ -2785,3 +2785,25 @@ pub fn ttyname(fd: RawFd) -> Result<PathBuf> {
     buf.truncate(nul);
     Ok(OsString::from_vec(buf).into())
 }
+
+/// Get the effective user ID and group ID associated with a Unix domain socket.
+///
+/// See also [getpeereid(3)](https://linux.die.net/man/3/getpeereid)
```

Comment:
This man page link dates from 2001!  Might be better to use a more up-to-date version, like https://www.freebsd.org/cgi/man.cgi?query=getpeereid

Reply:
Sounds good!